interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -785,6 +787,75 @@ public void deadlineInPast() throws Exception {
     }
   }
 
+  @Test(timeout = 10000)
+  public void maxInboundSize_exact() {
+    // warm up the channel and JVM
+    blockingStub.emptyCall(Empty.getDefaultInstance());
+    // 5 is the size of the encoded response proto
+    TestServiceGrpc.TestServiceBlockingStub stub = TestServiceGrpc.newBlockingStub(channel)
+        .withMaxInboundMessageSize(5);
```

Comment:
This is fragile, and if the response proto changes, it would be annoying to figure out the new size.

Some test methods construct golden responses. You could do the same and get the serialized size programatically. The actual response can also be compared with the golden response to make sure the response is of the expected size.